x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/analytics_detail_flyout/analytics_detail_flyout.tsx

```diff
@@ -0,0 +1,124 @@
+import { useGetAnalytics } from '../../../analytics_management/services/analytics_service';
+import type { AnalyticStatsBarStats } from '../../../../../components/stats_bar';
+import type { DataFrameAnalyticsListRow } from '../../../analytics_management/components/analytics_list/common';
+import { useMlLocator, useNavigateToPath } from '../../../../../contexts/kibana';
+import { ML_PAGES } from '../../../../../../locator';
+import { ExpandedRow } from '../../../analytics_management/components/analytics_list/expanded_row';
+
+export const AnalyticsDetailFlyout = () => {
+  const {
+    isDataFrameAnalyticsDetailsFlyoutOpen,
+    closeActiveFlyout,
+    setActiveJobId,
+    activeJobId: analyticsId,
+  } = useJobInfoFlyouts();
+  const [analytics, setAnalytics] = useState<DataFrameAnalyticsListRow[]>([]);
+  const [, setAnalyticsStats] = useState<AnalyticStatsBarStats | undefined>(undefined);
+  const [, setErrorMessage] = useState<any>(undefined);
+  const [, setIsInitialized] = useState(false);
+  const [, setJobsAwaitingNodeCount] = useState(0);
+  const blockRefresh = false;
+  const getAnalytics = useGetAnalytics(
+    setAnalytics,
+    setAnalyticsStats,
+    setErrorMessage,
+    setIsInitialized,
+    setJobsAwaitingNodeCount,
+    blockRefresh
+  );
+  const getAnalyticsCallback = useCallback(
+    (id: string | null) => getAnalytics(true, id),
+    [getAnalytics]
+  );
+
+  // Subscribe to the refresh observable to trigger reloading the analytics list.
+
+  useEffect(
+    function getAnalyticsDetailsOnChange() {
+      if (!analyticsId) {
+        return;
+      }
+      getAnalyticsCallback(analyticsId);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [analyticsId]
+  );
+  const analytic = useMemo(
+    () => analytics.find((a) => a.id === analyticsId),
+    [analytics, analyticsId]
+  );
+
+  const locator = useMlLocator()!;
+  const navigateToPath = useNavigateToPath();
+
+  const redirectToAnalyticsList = useCallback(async () => {
+    const path = await locator.getUrl({
+      page: ML_PAGES.DATA_FRAME_ANALYTICS_JOBS_MANAGE,
+    });
+    await navigateToPath(path, true);
+  }, [locator, navigateToPath]);
+
+  const flyoutTitleId = `mlAnalyticsDetailsFlyout-${analyticsId}`;
+  return isDataFrameAnalyticsDetailsFlyoutOpen ? (
+    <EuiFlyout
+      data-test-subj="analyticsDetailsFlyout"
+      type="overlay"
+      size="m"
+      ownFocus={false}
+      onClose={() => {
+        closeActiveFlyout();
+        setActiveJobId(null);
+      }}
+      aria-labelledby={flyoutTitleId}
+    >
+      <EuiFlyoutHeader hasBorder data-test-subj={`analyticsDetailsFlyout-${analyticsId}`}>
+        <EuiFlexGroup>
+          <EuiFlexItem grow={true}>
+            <EuiTitle size="s">
+              <h2 id={flyoutTitleId}>{analyticsId}</h2>
+            </EuiTitle>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty onClick={redirectToAnalyticsList} iconType="popout">
```

Comment:
Is the `popout` icon needed here as the jobs list is opening in the current page?

Reply:
Updated here [`466ddec` (#207141)](https://github.com/elastic/kibana/pull/207141/commits/466ddec987d94e12d4afef7df5b5f17fd4a85ea8)